src/platform/user/selectors.js

```diff
@@ -23,27 +20,15 @@ export const isMultifactorEnabled = state => selectProfile(state).multifactor;
 export const selectAvailableServices = state => selectProfile(state)?.services;
 export const selectPatientFacilities = state =>
   selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => {
-    // TODO: The work in this selector to override the `isCerner` values will be
-    // removed after the override logic gets moved to vets-api. ie, we will be
-    // able to trust the `isCerner` flags that come directly from vets-api.
-
     // Derive if the user belongs to a Cerner facility in the FE maintained list.
     const hasCernerFacilityID = CERNER_FACILITY_IDS.includes(facilityId);
 
-    // Derive if the feature toggle is on.
-    // TODO: can this feature toggle check be removed since it should always be true now?
-    const showNewScheduleViewAppointmentsPage =
-      state?.featureToggles?.[
-        featureFlagNames.showNewScheduleViewAppointmentsPage
-      ];
-
-    // Derive if they are a 200CRNR Cerner patient.
-    const isCernerPatient = selectProfile(state)?.isCernerPatient;
-
     // Derive if we should consider it a Cerner facility.
+    const isFlipperDisabled =
+      state?.featureToggles?.[`cerner_override_${facilityId}`] === false;
+    const isFlipperEnabled = !isFlipperDisabled;
```

Comment:
lol

Reply:
hmmm... could we do: `const isFlipperEnabled = state.featureToggles?.[`cerner_override_${facilityId}`] === true`?

Reply:
No because a bunch of unit tests would break from that very tiny difference

Reply:
It was difficult fixing the tests last time

Reply:
Basically, right now it is written to say "If we have a feature flag created and it is explicitly set to false, then do not allow that facility to pass through to Cerner systems."